pkgs/os-specific/linux/audit/default.nix

```diff
@@ -36,7 +36,7 @@ stdenv.mkDerivation rec {
   # TODO: Remove the musl patches when
   #         https://github.com/linux-audit/audit-userspace/pull/25
   #       is available with the next release.
-  patches = stdenv.lib.optional stdenv.hostPlatform.isMusl [
+  patches = [ ./allowed-permissions.patch ] ++ stdenv.lib.optional stdenv.hostPlatform.isMusl [
```

Comment:
Could you add a comment explaining the patch, ideally linking to a relevant upstream issue?

It would also be nice to move the musl comment inside the musl conditional to make it easier to see what it refers to.

Reply:
Did add a comment and reformatted the section.

Reply:
Could you also clarify whether there is an upstream issue or pull request?

Reply:
This still doesn't seem resolved.

Reply:
I don't think that there is any PR for linux-audit.

Reply:
Is there an issue at least? How will upstream learn we have this problem?

Reply:
I created a PR for audit there: https://github.com/linux-audit/audit-userspace/pull/138

I'm not sure how open they're about this contributions, so let's see.